### ISSUE TITLE
whento-agency-review

### ISSUE THREAD

**Issue by benante** [fac28/code-reviews#4]
### README

readme file empty

### User stories

User story missing

### Learning outcomes

[Does it demonstrate the learning outcomes for that week?]
Mostly, accessibility on html is complete, screen reader missing

### UI bugs

[Can you see any obvious bugs or areas to improve?]
"When to" button content overflowing parent div. 
Scroll function not working at the bottom of the page, preventing to submit the form.
Submit form accessibile only on mobile and working showing netlify website, even when not filled 

### Instructions

[Does everything work as expected or were there missing instructions?]

### File structure

[When you open the project in your editor, does the file structure make sense?]
Yes it does

### Flow of control

[ Can you you follow the different paths the code might take?]

### Naming

[Do variables and functions have clear and descriptive names?]
Yes 

### Readability

[Do you understand the code?]
Yes, it seems relatively easy to read
